### ISSUE TITLE
why i get this result ：No Answer: Answer(status=NO_ANSWER, chosenAnswer=null, images=null)

### ISSUE THREAD

**Issue by BWMa** [LarryDpk/Google-Bard#23]
11:01:06.817 [main] INFO com.pkslow.ai.GoogleBardClient -- getSNlM0e Response code: 200
11:01:07.555 [main] INFO com.pkslow.ai.GoogleBardClient -- calling Bard with request BardRequest{strSNlM0e=AOTFbH4txcjTYWlPABGtcQ_hFJcH:1693537266664, question=can you show me a picture of clock?, conversationId=, responseId=, choiceId=}
11:01:08.416 [main] INFO com.pkslow.ai.GoogleBardClient -- Ask Response code: 200
11:01:08.642 [main] DEBUG com.pkslow.ai.GoogleBardClient -- Result for ask: [["wrb.fr",null,null,null,null,[9]]]
11:01:08.642 [main] DEBUG com.pkslow.ai.GoogleBardClient -- Raw answers length: 36
11:01:08.681 [main] INFO com.pkslow.ai.GoogleBardExample -- No Answer: Answer(status=NO_ANSWER, chosenAnswer=null, images=null)

**Comment by kharemohit786** [comment]
@LarryDpk It is requiring to update the __Secure-1PSID and __Secure-1PSIDTS token every 15-20 minutes, cant think of deploying this on the server with this behavior, any workaround to fetch these tokens via OAuth or any persistent tokens?

**Comment by LarryDpk** [comment]
Yes, it's Google Bard restriction to us. I think they will provide the official API later. I didn't find a good soluction for this.

**Comment by saghassy** [comment]
Running into the same issue. Cookies expire every few minutes. Makes this almost useless. No workarounds for this?
